acceptance/chaos_test.go

```diff
@@ -272,13 +272,14 @@ CREATE TABLE bank.accounts (
 
 	// Verify accounts.
 
-	// Recreate db client because node 0 might have been restarted by
+	// Hold the read lock on node 0 to prevent it being restarted by
 	// chaos monkey.
-	db = makePGClient(t, c.PGUrl(0))
+	clients[0].RLock()
```

Comment:
does the monkey hold the lock until the node comes back?

can you put this below `var sum int`?